project_settings/settings.py

```diff
@@ -39,12 +40,14 @@
     "django.contrib.staticfiles",
     "django_prometheus",
     "bootstrap5",
+    "rest_framework_simplejwt",
```

Comment:
i guess we are allowed to use this `rest_framework_simplejwt`?
if you are not sure the add it to open questions